docs/guides/install-qiskit.mdx

```diff
@@ -236,6 +236,56 @@ Qiskit strives to support as many operating systems as possible, but due to limi
 
 Additionally, Qiskit only supports the CPython implementation of the Python language. Running with other Python interpreters such as PyPy is not supported.
 
+### Qiskit v2.x
+
+In the Qiskit v2.x release series the supported platforms are:
+
+<details>
+  <summary>
+    Tier 1
+  </summary>
+  Tier 1 operating systems are fully tested as part of the development processes to ensure any proposed change will function correctly. Pre-compiled binaries are built, tested, and published to PyPI as part of the release process. Typically, as long as there is a functioning Python environment installed, Qiskit can be installed on these operating systems without needing to install further dependencies.
+
+  Tier 1 operating systems:
+
+  - Linux x86_64 (distributions compatible with the [manylinux 2014](https://www.python.org/dev/peps/pep-0599/) packaging specification).
+  - macOS x86_64 (10.12 or later)
+  - macOS ARM64 (11.0 or newer)
+  - Windows 64-bit (Windows 10 and later supported)
+  - Linux AArch64 (distributions compatible with the [manylinux 2014](https://www.python.org/dev/peps/pep-0599/) packaging specification)
+</details>
+
+<details>
+  <summary>
+    Tier 2
+  </summary>
+  Tier 2 operating systems are not tested as part of development process. However, pre-compiled binaries are built, tested, and published to PyPI as part of the release process and these packages can be expected to be installed with only a functioning Python environment.
+
+  Tier 2 operating systems:
+
+  -
```

Comment:
Is this a typo?

Reply:
Do you mean, the fact that the list of tier 2 operating systems isn't there? FYI @mtreinish

Reply:
There aren't any tier 2 platforms on Qiskit 2.0.0, the only one from 1.x (aarch64 linux) was promoted to tier 1 in 2.0.0. I left it here as an empty list to indicate this, but I'm fine with whatever formatting makes sense here